src/SFA.DAS.ApplyService.Web/Services/RoatpOrganisationVerificationService.cs

```diff
@@ -0,0 +1,72 @@
+﻿using SFA.DAS.ApplyService.Application.Apply.Roatp;
+using SFA.DAS.ApplyService.Domain.Entities;
+using SFA.DAS.ApplyService.Domain.Roatp;
+using SFA.DAS.ApplyService.Session;
+using SFA.DAS.ApplyService.Web.Infrastructure;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.ApplyService.Web.Services
+{
+    public class RoatpOrganisationVerificationService : IRoatpOrganisationVerificationService
```

Comment:
Couldn't see any specific tests for this. Do they exist?

Reply:
Added tests for this service